### PR TITLE
Add new Dev UI card

### LINKED ISSUE #208: Migrate to the new Dev UI
https://github.com/quarkusio/quarkus/issues/31315

### PR DESCRIPTION
**Pull request by phillip-kruger**
Screenshot_20230620_175823

Fix #208

### CODE REVIEW

**Comment by phillip-kruger** [comment]
@jtama please have a look

File: nbactions.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jtama) [inline nbactions.xml:1]: What is this file for ?

Comment (phillip-kruger) [reply nbactions.xml:1]: a, crap, that should not be there. Hold on let me fix

**Comment by jtama** [comment]
LGTM.

WIll merge as soon as the build is green.

**Review by jtama — APPROVED** [review]

**Pull request merged**